electrumabc_gui/qt/consolidate_coins_dialog.py

```diff
@@ -245,7 +259,7 @@ def __init__(self, parent=None):
         )
 
         self.maximum_height_sb = BlockHeightSpinBox()
-        self.maximum_height_sb.setValue(1_000_000)
+        self.maximum_height_sb.setValue(10_000_000)
```

Comment:
The best value is the current tip, or current tip + 1, but that will prevent accidental misuses for now